docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -49,9 +49,12 @@ SELECT queries
 DQL SELECT clause
 ~~~~~~~~~~~~~~~~~
 
-The select clause of a DQL query specifies what appears in the
-query result. The composition of all the expressions in the select
-clause also influences the nature of the query result.
+The SELECT clause of a DQL query specifies what gets hydrated in
+the query result. You are always returned usable objects, but any
+associated objects not included in the SELECT clause will be
+proxies (ie. unhydrated).  They get hydrated by Doctrine when
```

Comment:
Possibly, but remember that you are approaching readers that may have no clue about doctrine, so this stuff can be plain confusing until the basics are covered :-)